packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

```diff
@@ -22,7 +22,7 @@ The [Brands API](/docs/reference/api/brands/) allows you to set all of the follo
 
 ## Before you begin
 
-You need to make sure that you have an [Okta organization](/docs/guides/quickstart/cli/create-org/) set up to test this functionality, and you need to request access to the Brands API from [Okta support](https://support.okta.com/help).
+You need to make sure that you have an [Okta organization](http://localhost:8080/docs/guides/quickstart/cli/main/#create-your-okta-organization) set up to test this functionality, and you need to request access to the Brands API from [Okta support](https://support.okta.com/help).
```

Comment:
Why are you adding http://localhost:8080/ to this url?

```suggestion
You need to make sure that you have an [Okta organization](http://localhost:8080/docs/guides/quickstart/cli/main/#create-your-okta-organization) set up to test this functionality, and you need to request access to the Brands API from [Okta support](https://support.okta.com/help).
```

Reply:
oops, those localhost URLs were a mistake. Your suggestions fixes most of them, but I've fixed this one. Thanks!